src/components/ContentWidgets/Homepage/HomepageEditor.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FormEvent, ChangeEvent } from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import './Homepage.css';
+
+export const HomepageEditor: React.FC<WidgetEditorProps> = ({
+    originalContent, 
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    let homepage_content = editedContent.homepage_content || 
+        {title: "Title not set", subtitle: "Subtitle not set", backgroundImage: "", blur: "", bgLightness: "", bgOpacity: ""};
```

Comment:
have this object be put into a constant `defaultHomepageContent` or something and also split it up for one property per line eg

```
{
  title: "",
  subtitle: ""
  ...
}
```